HW6.py

```diff
@@ -0,0 +1,224 @@
+        be printed
+        """
+        job_result = master.try_to_fix(self, refr)
+        print('{} called to {}.'.format(self.name, master.name), end=' ')
+        print('He fixed it' if job_result else 'He can\'t fix it')
+
+    def earn_money(self, salary=200):
+        """Parent get some money after this method"""
+        self.money += salary
+
+
+class Child(User):
+    """Child is user who can hit refrigerator"""
+    def hit_fridge(self, refr):
+        """After hit refriger. will be broken with a thirty percent chance """
+        if random.random() < 0.3:
+            refr.broken = True
+            print('{} broke {}!'.format(self.name, refr.name))
+        else:
+            print('{} hit {}, but not broke.'.format(self.name, refr.name))
+
+
+class Master(object):
+    """Master is a person who can fix refrigerators if he have enough skill"""
+    money = 100
+
+    def __init__(self, skill, name='Master', company='own'):
+        self.skill = skill
+        self.name = name
+        self.company = company
+
+    def advertisement(self):
+        """Advertising of master. Skills always increase by two"""
+        print('Highly skilled master {0} from {1} company is looking for work!'
+              ' My skill is {2}. Prices:\nImport refrigerator: {3}\nDomestic '
+              'refrigerator: {4}'.format(self.name, self.company,
+                                         str(self.skill + 2),
+                                         str(self.skill * 150),
+                                         str(self.skill * 100)))
+
+    def try_to_fix(self, customer, refr):
+        """Master will try to fix refrigerator
+
+        Price formula: skill * 100 * сoeff
+        Netherlands refrigerators can be fixed by only masters with >=7 skill,
+        coeff is 1,5
+        Belarus refrigerators can be fixed only by masters with >=3 skill,
+        coeff is 1, and also help to improve some skills.
+        Payment is made by the client (Parent)
+        Сonsultation is free.
+        """
+        if refr.made_in == 'The Netherlands':
+            if self.skill >= 7:
+                refr.broken = False
+                customer.money -= self.skill * 100 * 1.5
+                return True
+        elif refr.made_in == 'Belarus':
+            if self.skill >= 3:
+                refr.broken = False
+                customer.money -= self.skill * 100 * 1
+                self.skill += 0.5
+                return True
+        return False
+
+    def study(self):
+        """Master can pay money and improve his skills"""
+        if self.money >= 100:
+            self.money -= 100
+            self.skill += 1
+        else:
+            print('{self.master} don\'t have enough money to study')
+
+
+class Product(object):
+    """Product is some food we can eat or put to refrigerator
+
+    It can be cold and freeze type.
+    Every product aso have size -oit shows how many cells this product need
+    """
+    in_cold_cam = None
```

Comment:
When I'll put products would be created instance variable for each instance. 
the only use of these vars are not to take exception in your if-elif condition. it should be refactored.

Reply:
Fixed.
Now instead of "in_cold_cam" and "in_freeze_cam" it is only one product property - location.